src/app/containers/PageHandlers/withOptimizelyProvider/index.jsx

```diff
@@ -5,14 +5,12 @@ import {
   setLogger,
 } from '@optimizely/react-sdk';
 import { ServiceContext } from '#contexts/ServiceContext';
-import isLive from '#lib/utilities/isLive';
 import { GEL_GROUP_3_SCREEN_WIDTH_MAX } from '@bbc/gel-foundations/dist/breakpoints';
 import onClient from '#lib/utilities/onClient';
 import getOptimizelyUserId from './getOptimizelyUserId';
 
-if (isLive()) {
-  setLogger(null);
-}
```

Comment:
I wonder if its worth keeping this in for local dev / Test, as its quite useful to see the logging when being bucketed into experiments. Is it possible to check if its within the Cypress environment to also just disable it there?

Reply:
Tried adding the cypress environment check but doesn't look like it is working unfortunately. I'll check to see if any other environment check can be used instead.